Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml

```diff
@@ -248,6 +248,14 @@
       factions:
         - Syndicate
     - type: Access
+      tags:
+       - NuclearOperative
+       - SyndicateAgent
+    - type: Lock
+      locked: true
+    - type: ActivatableUIRequiresLock
```

Comment:
Both NT and syndicate have this set.

Reply:
yes, they both have it because the base borgs dont have those components

Reply:
if i remove that set itll not have a locking mechanism anymore

Reply:
could u elaborate more

Reply:
I think sloth means that since both syndi and NT borgs have this set AND they are the only things inheriting from BaseBorgChasis, it would be better to remove this set from BOTH of them and add it to the BaseBorgChasis prototype

Reply:
I think sloth means that since both syndi and NT borgs have this set AND they are the only things inheriting from BaseBorgChasis, it would be better to remove this set from BOTH of them and add it to the BaseBorgChasis prototype

Reply:
alright

Reply:
hopefully resolved